packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.tsx

```diff
@@ -115,8 +124,15 @@ function DefaultValueListField(
     <MultiValueField
       value={value}
       onChange={arr => {
-        // Convert null to undefined
-        onChange(arr || undefined)
+        if (arr == null) {
+          // Convert null to undefined
+          onChange(undefined)
+        } else if (columnModel.columnType === ColumnTypeEnum.DATE_LIST) {
+          onChange(arr.map(val => dayjs(val).valueOf()))
```

Comment:
Change behavior to match what is passed in - a list of integer timestamps

Reply:
Just curious, why does this `dayjs` call not need the 'x' argument to indicate parsing the value as a ms Unix Timestamp?

Reply:
In this case, these are ISO timestamps. I'll add a comment to clarify